### PR TITLE
remove reference to samples folder in project structure doc

### PR DESCRIPTION
**Pull request by ddsharpe**
The sample folder was removed from the source structure but the reference in the documentation was still there.

### CODE REVIEW

**Review by rakillen — APPROVED** [review]

File: documentation/2.0/content/developer/project-structure.md [hunk 1]
```diff
@@ -5,14 +5,16 @@ draft: false
...
+
+The `system-test` and `alias-test` modules contain test suites for verifying pull requests and nightly regression testing.
 
 Alias definitions are discussed in more detail [here]({{< relref "/developer/alias-definitions.md" >}}).
 
-The `installer` module builds the final installer ZIP file. It includes the assembly definitions, start scripts for each tool for Linux and Windows platforms, and configurations for variable injection and logging.
+The `installer` module builds the final installer ZIP file. `installer` includes the assembly definitions, start scripts for each tool for Linux and Windows platforms, and configurations for variable injection and logging.
 
-A single installer ZIP file is built under the `WLSDEPLOY_HOME/installer/target` directory.
+Two installer files are built under the `WLSDEPLOY_HOME/installer/target` directory, one ZIP for all platforms, and one tar.gz for those that prefer tar.gz on Unix systems.
```
Comment (rosemarymarano) [inline documentation/2.0/content/developer/project-structure.md:18]: one ZIP for all platforms, and one tar.gz for those that prefer tar.gz on Unix systems. -> one ZIP file for all platforms, and one `tar.gz` file for those that prefer `tar.gz` files on UNIX systems.

File: documentation/2.0/content/developer/project-structure.md [hunk 2]
```diff
@@ -5,14 +5,16 @@ draft: false
...
+
+The `system-test` and `alias-test` modules contain test suites for verifying pull requests and nightly regression testing.
 
 Alias definitions are discussed in more detail [here]({{< relref "/developer/alias-definitions.md" >}}).
 
-The `installer` module builds the final installer ZIP file. It includes the assembly definitions, start scripts for each tool for Linux and Windows platforms, and configurations for variable injection and logging.
+The `installer` module builds the final installer ZIP file. `installer` includes the assembly definitions, start scripts for each tool for Linux and Windows platforms, and configurations for variable injection and logging.
 
-A single installer ZIP file is built under the `WLSDEPLOY_HOME/installer/target` directory.
+Two installer files are built under the `WLSDEPLOY_HOME/installer/target` directory, one ZIP file for all platforms, and one tar.gz file for those that prefer the tar.gz format on UNIX systems.
```
Comment (rosemarymarano) [inline documentation/2.0/content/developer/project-structure.md:18]: tar.gz file -> `tar.gz` (needs code font)
tar.gz format -> `tar.gz` (needs code font)

**Review by rosemarymarano — COMMENTED** [review]
Minor edits needed

**Review by rosemarymarano — APPROVED** [review]
Perfect. Thank you.

**Review by robertpatrick — APPROVED** [review]

**Pull request merged**